config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -119,6 +119,26 @@ presubmits:
     annotations:
       testgrid-dashboards: presubmits-kops
       testgrid-tab-name: verify-bazel
+  - name: pull-kops-verify-gomod
+    branches:
+    - master
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - ./hack/verify-gomod
```

Comment:
Can you point me to the repo that has `./hack/verify-gomod`?

Reply:
https://github.com/kubernetes/kops/blob/master/hack/verify-gomod

Reply:
It's there now!  It might not have been when I first opened the PR :-)

https://github.com/kubernetes/kops/blob/master/hack/verify-gomod